wiki/Tournaments/CWC/1/en.md

```diff
@@ -5,149 +5,149 @@ tags:
 
 # Catch the Beat World Cup
 
-The **Catch the Beat World Cup** (***CWC***) was a team-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It was the first installment of the osu!catch World Cup.
+The **Catch the Beat World Cup** (***CWC***) was a single elimination team-based osu!catch tournament organized by various osu! community members. Despite not being hosted by (or being run under the direct provision of) the [osu!team](/wiki/People/The_Team), it was formally considered as the first installment of the osu!catch World Cup tournament series.
```

Comment:
```suggestion
The **Catch the Beat World Cup** (***CWC***) was a single elimination, team-based [osu!catch](/wiki/Game_mode/osu!catch) tournament organized by various osu! community members. Despite not being hosted by (or being run under the direct provision of) the [osu!team](/wiki/People/The_Team), it was formally considered as the first installment of the osu!catch World Cup tournament series.
```

Reply:
i'll pass this i think since on the other world cup articles "osu!team" is the only one that got attached to a link

![image](https://user-images.githubusercontent.com/36564236/106345777-29029500-62e5-11eb-8e3d-30f299c0041e.png)

![image](https://user-images.githubusercontent.com/36564236/106345797-45063680-62e5-11eb-883c-500205bd9ae3.png)